protected/modules/OphCoCorrespondence/components/OphCoCorrespondence_API.php

```diff
@@ -566,7 +566,8 @@ public function getAddress($patient_id, $contact_string, $nickname = false)
         if (!$text_ElementLetter_address) {
             $text_ElementLetter_address = '';
         }
-        
+        $address = ($contact->contact->label?$contact->contact->label->name:'')."\n".$address;
```

Comment:
I think the newline should be in the ternary operation, not outside it. We don't want the newline if there is no role specified.